### DISCUSSION TITLE
Deprecate Node3D's `global_scale`, rename it to `apply_scale` like Node2D

### DISCUSSION THREAD

**Discussion by Mickeon** [godotengine/godot-proposals#9257] [3D]
Have you noticed Node3D's `global_scale` is a method? No, it's not `scale` but global. And no, it does not follow the other similarly named properties prefixed with `global_`.

But wait. Node2D does in fact have `global_scale`, and a similar method is instead named `apply_scale`.

The suggestion here is too trivial to be a proposal, so I'd like some extra thoughts on it here.

**Comment by AThousandShips** [comment]
That would break the structure, so this should also involve renaming `global_rotate` and `global_translate`, they should probably be renamed to `trabslate/rotate/scale_global` instead

You also have `Node2D.global_translate` so it's not unique to `Node3D`

**Reply by AThousandShips** [reply]
I don't think this is critical enough to add clutter with duplicate methods though
